internal/commands/server/command.go

```diff
@@ -60,6 +62,7 @@ type Command struct {
 	flagConsulDestinationNamespace string
 	flagMirrorK8SNamespaces        bool
 	flagMirrorK8SNamespacePrefix   string
+	flagPartitionInfo              consul.PartitionInfo
```

Comment:
Just FYI, the addition of these flags means that previous versions of the controller will no longer work if we make a corresponding k8s change (since this is an unknown flag to them), so it's a breaking change. Personally just to make it so we don't really have to think about the support matrix for this, I'd prefer if we just passed these as environment variables.

Reply:
move these to env variables